### PR TITLE
termius: unbreak

### LINKED ISSUE #129836: termius: requested URL returned error 403
**Describe the bug**
`termius` cannot be installed because the requested URL returned error 403.

**To Reproduce**
`nix-env -iA nixos.termius`

**Expected behavior**
`termius` should be installed.

**Screenshots**
Relevant logs are as below:
```
building '/nix/store/2q6jql1vd8hgfpw1rbwncra1y8ffqscv-unit-nix-daemon.service.drv'...

trying https://deb.termius.com/pool/main/t/termius-app/termius-app_7.1.0_amd64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
curl: (22) The requested URL returned error: 403
error: cannot download termius-app_7.1.0_amd64.deb from any mirror
```

**Additional context**

**Notify maintainers**
@Br1ght0ne @misuzu @Th0rgal 

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.45, NixOS, 21.05.1353.c06613c25df (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.12`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module: 
```

### PR DESCRIPTION
**Pull request by Artturin**
###### Motivation for this change
Closes  #129836

###### Things done
switched src to use a web.archive.org link

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Th0rgal — APPROVED** [review]
lgtm

**Comment by Artturin** [comment]
changed the version to the correct one in the latest push

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 129868` at 3cb219aa run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>termius</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/applications/networking/termius/default.nix:44:3:
```
   |
44 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/applications/networking/termius/default.nix [hunk 1]
```diff
@@ -12,11 +12,12 @@
 
 stdenv.mkDerivation rec {
   pname = "termius";
-  version = "7.1.0";
+  version = "7.15.1";
 
   src = fetchurl {
-    url = "https://deb.termius.com/pool/main/t/termius-app/termius-app_${version}_amd64.deb";
```
Comment (misuzu) [inline pkgs/applications/networking/termius/default.nix:18]: It's `https://deb.termius.com/pool/main/t/termius-app/termius-app_7.16.0_amd64.deb` now.
See `wget -qO- https://deb.termius.com/dists/squeeze/main/binary-amd64/Packages`.

Comment (Artturin) [reply pkgs/applications/networking/termius/default.nix:18]: Thanks! i did not know that command

**Pull request merged**